jvb/src/test/kotlin/org/jitsi/videobridge/cc/BitrateControllerTest.kt

```diff
@@ -82,13 +103,206 @@ class BitrateControllerTest : ShouldSpec() {
                 )
             }
         }
+
+        context("Allocation") {
```

Comment:
I suggest this should have tests for Last-N, audio-only, and pinning modes, too?